reskindex.js

```diff
@@ -55,28 +28,38 @@ strm.write(" * so you'd just be trying to swim upstream like a salmon.\n");
 strm.write(" * You are not a salmon.\n");
 strm.write(" */\n\n");
 
-var mySkinfo = JSON.parse(fs.readFileSync(skinfoFile, "utf8"));
-
-strm.write("var skin = {};\n");
-strm.write('\n');
+if (packageJson['matrix-react-parent']) {
+    strm.write("module.exports.components = require('"+packageJson['matrix-react-parent']+"/lib/component-index').components;\n\n");
+} else {
+    strm.write("module.exports.components = {};\n");
+}
 
-var files = glob.sync('**/*.js', {cwd: viewsDir});
+var files = glob.sync('**/*.js', {cwd: componenetsDir});
 for (var i = 0; i < files.length; ++i) {
     var file = files[i].replace('.js', '');
+
     var module = (file.replace(/\//g, '.'));
```

Comment:
This ends up shadowing the `module` variable which is defined for every file in Node. It also makes syntax highlighting a pain. Can we rename it please? :angel: